aries-site/src/layouts/main/Layout.js

```diff
@@ -23,6 +23,7 @@ export const Layout = ({
         {size => (
           <Box
             // pad={{ horizontal: calcPad(size) }}
+            height={{ min: '100vh' }}
```

Comment:
nice.

Reply:
Should we do something similar for the SidebarLayout?

Reply:
Yeah we can look into seeing if the same thing would work.